openfeature/event_executor_test.go

```diff
@@ -160,7 +160,7 @@ func TestEventHandler_Eventing(t *testing.T) {
 
 		// trigger event from provider implementation
 		eventingImpl.Invoke(Event{
-			ProviderName: eventingProvider.Metadata().Name,
+			ProviderName: eventingProvider.Metadata().Domain,
```

Comment:
This should remain provider name but the event could be extended to include a domain property.